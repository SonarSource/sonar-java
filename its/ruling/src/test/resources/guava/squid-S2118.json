{
'com.google.guava:guava:src/com/google/common/collect/AbstractBiMap.java':[
383,
],
'com.google.guava:guava:src/com/google/common/collect/ConcurrentHashMultiset.java':[
563,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSetMultimap.java':[
479,
],
'com.google.guava:guava:src/com/google/common/collect/Multimaps.java':[
136,
137,
215,
216,
293,
294,
377,
378,
],
'com.google.guava:guava:src/com/google/common/collect/TreeMultimap.java':[
248,
249,
],
'com.google.guava:guava:src/com/google/common/collect/TreeMultiset.java':[
960,
],
}
