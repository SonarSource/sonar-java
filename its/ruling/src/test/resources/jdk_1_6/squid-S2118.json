{
'jdk_1_6:java/util/TreeSet.java':[
485,
],
}
