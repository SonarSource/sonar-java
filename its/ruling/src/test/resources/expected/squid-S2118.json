{
'project:java/util/TreeSet.java':[
502,
],
'project:javax/management/remote/rmi/RMIConnectorServer.java':[
727,
],
'project:javax/swing/LayoutFocusTraversalPolicy.java':[
255,
],
}
